torch/autograd/gradcheck.py

```diff
@@ -201,7 +201,7 @@ def compute_numerical_jacobian_cols(jvp_fn, delta, input_is_complex) -> List[tor
     ds_dx_tup = jvp_fn(delta)
 
     if input_is_complex:  # C -> R
-        ds_dy_tup = jvp_fn(delta * 1j)
+        ds_dy_tup = jvp_fn(delta * 1j) if delta_i is None else jvp_fn(delta_i * 1j)
```

Comment:
Slow version does not use delta_i. But now I've changed it to use a single parameter delta object which may be either tuple or tensor or python number. It will be always python number in the slow case, and tuple or tensor in the fast case depending on whether that input is complex.